packages/fast-components-react-base/src/tabs/tabs.tsx

```diff
@@ -142,14 +162,59 @@ class Tabs extends Foundation<TabsHandledProps, TabsUnhandledProps, TabsState> {
         return React.Children.map(this.tabItems(), this.renderTabItem);
     }
 
+    private setActive(tabItem: TabItems, index: number): boolean {
+        if (this.state.activeId) {
+            return this.state.activeId === tabItem.id;
+        } else {
+            return index === 0;
+        }
+    }
+
     private tabItems(): React.ReactNode {
-        return this.getChildrenBySlot(
-            this.props.children,
-            this.getSlot(TabsSlot.tabItem)
-        );
+        if (this.props.tabItems) {
+            return this.props.tabItems.map((tabItem: TabItems, index: number) => {
+                return (
+                    <TabItem key={tabItem.id} id={tabItem.id} slot={TabsSlot.tabItem}>
+                        <Tab
+                            {...tabManagedClasses}
+                            slot={TabsSlot.tab}
+                            key={tabItem.id}
+                            aria-controls={tabItem.id}
+                            active={this.setActive(tabItem, index)}
+                            onClick={this.handleClick}
+                            onKeyDown={this.handleKeyDown}
+                            tabIndex={this.setActive(tabItem, index) ? 0 : -1}
+                        >
+                            {tabItem.tab("")}
+                        </Tab>
+                        <TabPanel
+                            {...tabPanelManagedClasses}
+                            slot={TabsSlot.tabPanel}
+                            key={tabItem.id}
+                            id={tabItem.id}
+                            aria-labelledby={tabItem.id}
+                            active={this.setActive(tabItem, index)}
+                        >
+                            {tabItem.content("")}
```

Comment:
same issue here, shouldn't this pass a string for the class name?

Reply:
Yup!